src/midi/midi_lasynth_model.cpp

```diff
@@ -49,65 +50,91 @@ LASynthModel::LASynthModel(const std::string &rom_name,
 	assert(ctrl_full || (ctrl_a && ctrl_b));
 }
 
-// Checks if its ROMs can be positively found in the provided directory
-bool LASynthModel::InDir(const service_t& service, const std_fs::path& dir) const
+// TODO: Possibly add caching to this function in persistant storage (perhaps a
+// file in ~/.config/dosbox) The identifyROMFile method in the mt32emu library
+// compares with known SHA1 sums so it may be slow for users with lots of files
+// to search. This is mitaged by checking the file size first at least, only
+// running a checksum on files of the correct size.
+std::optional<std_fs::path> LASynthModel::find_rom(const service_t& service,
```

Comment:
Given the timings are really good, I think we can remove this TODO comment as it's fast enough to detect all the ROMs.